minari/dataset/minari_dataset.py

```diff
@@ -255,9 +213,15 @@ def filter_episodes(self, condition: Callable[[h5py.Group], bool]) -> MinariData
         ```
 
         Args:
-            condition (Callable[[h5py.Group], bool]): callable that accepts an episode group and returns True if certain condition is met.
+            condition (Callable[[Any], bool]): callable that accepts any type(For our current backend, an h5py episode group) and returns True if certain condition is met.
```

Comment:
Why not `Callable[EpisodeData]`?

And then the comment updated accordingly (it should not mention h5py): `callable that accepts EpisodeData and returns True if certain condition is met` or `boolean function on EpsiodeData`

Reply:
done